lib/lita/adapters/slack.rb

```diff
@@ -38,9 +38,28 @@ def roster(target)
         room_roster target.id, api
       end
 
-      def send_messages(target, strings)
+      # @param [Array] messages list of String messages or Symbol emoji reactions
+      # messages starting with the ellipsis character will start a new thread
+      def send_messages(target, messages)
         api = API.new(config)
-        api.send_messages(channel_for(target), strings)
+        channel = channel_for(target)
+        timestamp = try_get(target, :timestamp)
+        thread_ts = try_get(target, :thread_ts)
```

Comment:
I feel like you could inline them, it's not that bad:
```ruby
timestamp = target.timestamp if target.respond_to?(:timestamp)
thread_ts = target.thread_ts if target.respond_to?(:thread_ts)
```

Also why would the channel not be a SlackSource?

Reply:
Ah I forgot about respond_to, switched to use that.

I wasn't sure how the message decoding worked, i.e. if it always got a `SlackSource` or if it used the default one without the `thread_ts` and `timestamp` if those weren't defined.

Reply:
So you changed the creation of Source by SlackSource, so we shouldn't need to be defensive about this here. Maybe I'm misunderstanding something, but it's the only place we create a `Lita::Source` in this repo.